src/common/low_precision_transformations/src/recurrent_cell.cpp

```diff
@@ -0,0 +1,237 @@
+                                                                  dataPrecision.max,
+                                                                  dataPrecision.hasZeroPoint,
+                                                                  updatePrecisions);
+                std::shared_ptr<ngraph::Node> new_fq = std::get<0>(QDQ);
+                std::shared_ptr<ngraph::Node> deq_multiply = std::get<1>(QDQ);
+                auto multiply_parent = deq_multiply->get_input_node_shared_ptr(0);
+                ov::disable_constant_folding(multiply_parent);
+                propagateSkipCleanupAttribute(deq_multiply);
+                this->register_new_node(new_fq);
+                updateOutput(context, deq_multiply, new_fq);
+            } else {
+                continue;
+            }
+        } else {
+            if (is_type<ngraph::opset1::Multiply>(lstm_parent)) {
+                auto multiply = lstm_parent->get_input_node_shared_ptr(0);
+                ov::disable_constant_folding(multiply);
+                propagateSkipCleanupAttribute(lstm_parent);
+            }
+            continue;
+        }
+    }
+    return true;
+}
+
+bool RecurrentCellTransformation::canBeTransformed(const TransformationContext& context, std::shared_ptr<Node> lstm) const {
+    std::shared_ptr<ov::Node> W, R;
+
+    if (is_type<opset5::LSTMSequence>(lstm)) {
+        W = lstm->get_input_node_shared_ptr(4);
+        R = lstm->get_input_node_shared_ptr(5);
+    } else if (is_type<opset5::GRUSequence>(lstm)) {
+        W = lstm->get_input_node_shared_ptr(3);
+        R = lstm->get_input_node_shared_ptr(4);
+    } else {
+        THROW_TRANSFORMATION_EXCEPTION << "Unexpected Operation type " << lstm;
+    }
+
+    for (auto fq_on_weight : {W, R}) {
+        if (auto fq_node = as_type_ptr<ngraph::opset1::FakeQuantize>(fq_on_weight)) {
+            const QuantizationDetails quantizationDetails = QuantizationDetails::getDetails(fq_node);
+            const auto precisionsAttribute = getAttributeFromOutput<PrecisionsAttribute>(fq_on_weight);
+            const auto precisions = precisionsAttribute.empty()
+                ? defaultPrecisions
+                : precisionsAttribute.as<PrecisionsAttribute>().value();
+            const DataPrecision dataPrecision = getDataPrecision(fq_on_weight, quantizationDetails, precisions);
+            auto QDQ = NetworkHelper::decomposeFakeQuantize(fq_node,
+                                                            dataPrecision.precision,
+                                                            dataPrecision.min,
+                                                            dataPrecision.max,
+                                                            dataPrecision.hasZeroPoint,
+                                                            updatePrecisions);
+            std::shared_ptr<ngraph::Node> new_fq = std::get<0>(QDQ);
+            std::shared_ptr<ngraph::Node> deq_multiply = std::get<1>(QDQ);
```

Comment:
Looks like we have the same decomposition code in transformation part. I would suggest to move decomposed fq checks to the transformation code to reduce impact on the load time.

Reply:
Done. Please double check